tlair/src/test/scala/at/forsyte/apalache/tla/lir/TestModule.scala

```diff
@@ -16,16 +16,16 @@ class TestModule extends AnyFunSuite {
   test("AlternatingBit module from Lamport's book") {
```

Comment:
This whole test suite probably needs refactoring. It only constructs value without having any assertions. While this is useful for seeing that the code compiles as expected, we could add some useful tests too.

Reply:
Yes, I'd meant to ask what the intent with this test was. I figured it was only testing for compilation without crashing. I'll creat an issue to investigate and improve (or remove?) this test.